### ISSUE TITLE
Excellent format

### ISSUE THREAD

**Issue by mml1** [Emanny58/colmar#3]
Your code is clean. You have comments, consistent indentation, and great naming conventions. However, in terms of naming conventions, I think your choice of articlez and articlle could be improved. It is not immediate clear what these divs are for.

```
   <div class="articlez">
        <div class="articlle grey">
```

https://github.com/Emanny58/colmar/blob/master/index.html#L208-L209

**Comment by Emanny58** [comment]
yeah it really isn't that efficient it was just i copied the section from the information div and therefore had to improvise. but thank you for the Specific feedback it's really appreciated.
